docs/theming/developers.md

```diff
@@ -272,14 +280,15 @@ Context consumers are elements that passively consume the parent context.
 Elements such as `<rh-cta>`, `<rh-badge>` and `<rh-tag>` are examples of
 consumers.
 
-To make your element a color context consumer,
+To make your element a color context consumer:
 
-1. First `import` the classMap Lit directive, and the consumer controller
+1. First `import` the [classMap Lit directive](https://lit.dev/docs/templates/directives/#classmap),
+   and the consumer controller.
 2. Then add the `@colorContextConsumer()` decorator to a private property of
```

Comment:
`on which` → `on, which`

Reply:
Not seeing `on which` in this diff. Where are you suggesting this change?